### PR TITLE
Tiddlywiki com - Update to the Community page

### PR DESCRIPTION
**Pull request by OdinJorna**
Link to forum post: https://groups.google.com/g/tiddlywiki/c/hxEcc06UBhY

I went through all the entries on the community page to bring this up to date. Goal is to bring more organization to make it easier for people to find stuff to enhance their wiki with.
I did the following:
1) Split the 'resources' tab into categories (Community plugins, Community themes, Community palettes, Community Editions, Other Resources)
2) Removed entries with dead links (and one entry on by request)
3) Added entries based on the '[best of 2020 forum thread'](https://groups.google.com/g/tiddlywiki/c/f23Eof4TTHI/m/m5ShOM7rBQAJ)
4) Went through all existing entries and sorted them into the new categories by changing the tags and moving them into new folders.

This is my first time using git outside of tutorials, so hopefully everything went as it should. Let me know if there are any thoughts, or mistakes I made that I can fix..

### CODE REVIEW

**Comment by Jermolene** [comment]
Thanks @OdinJorna. The way that you've canvassed opinions on the main group and then condensed them into a clean PR with a clear description is much appreciated. There's a lot of scope to improve the docs and we warmly welcome new contributors to this area.

File: editions/tw5.com/tiddlers/community/Community.tid [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-modified: 20140911084937570
+modified: 20210101170443584
 tags: TableOfContents
 title: Community
 type: text/vnd.tiddlywiki
 
 Here we gather the latest and most useful material from the TiddlyWiki community.
 
-<<tabs "Forums Latest Tutorials Resources Examples Articles Meetups" "Latest">>
+<<tabs "Forums Latest Tutorials [[Community Editions]] [[Community Plugins]] [[Community Themes]] [[Community Palettes]] [[Other resources]] Examples Articles Meetups" "Latest">>
```
Comment (pmario) [inline editions/tw5.com/tiddlers/community/Community.tid:9]: Typo: It should be [[Other Resources]]    ... see the capital "R"

**Comment by kookma** [comment]
@OdinJorna 
Many thanks for all your efforts. A minor issue very old plugins now appears on the top! I think you should disable timestamp.

**Comment by Jermolene** [comment]
> @OdinJorna 
> 
> Many thanks for all your efforts. A minor issue very old plugins now appears on the top! I think you should disable timestamp. 

Ouch, it would be good to fix that if feasible.

**Comment by OdinJorna** [comment]
ah yes. I edited the tag on each tiddler so the modified field was updated. Maybe it can sort on date created instead of date modified? I can see if i can revert the old values. I'll see what is possible when I have some time for another update.

**Comment by Jermolene** [comment]
Hi @OdinJorna no worries the modified dates are now fixed in eadbd62e6d694fc25fc6fa94a2632cc51d874bb4

I did it quite a hacky way:

1. Visit https://tiddlywiki.com/prerelease and copy the titles of the tiddlers modified on 1st January 2021 from the "Recent" tab
2. In a text editor reformat the titles as a 97 element array
3. In the browser console, run the first snippet below
4. Visit https://tiddlywiki.com
5. In the browser console, run the second snippet

Snippet to get the modified dates:

```js
var result=[];
[
"\"Drift - Collect, Organise, Grow.\" by Tony K",
 <snip>,
"Community Editions","Community Plugins"].forEach(title => {
var tiddler = $tw.wiki.getTiddler(title);
if(tiddler) {
result.push([title,tiddler.getFieldString("modified")])	
}
});
copy(result)
```

Snippet to write the modified dates:

```js
[
<array contents copied from previous snippet>
].forEach(([title,modified]) => {
var tiddler = $tw.wiki.getTiddler(title);
if(tiddler) {
$tw.wiki.addTiddler(new $tw.Tiddler(tiddler,{modified: modified}))
}
});
```

**Pull request merged**